crates/kitsune_p2p/dht/Cargo.toml

```diff
@@ -34,7 +34,7 @@ proptest-derive = { version = "0", optional = true }
 kitsune_p2p_dht = { path = ".", features = ["test_utils", "sqlite"]}
 
 kitsune_p2p_dht_arc = { path = "../dht_arc", features = ["test_utils"]}
-holochain_serialized_bytes = "0.0.53"
+holochain_serialized_bytes = "=0.0.53"
```

Comment:
Everywhere else in the project we've pinned this, so here I get a warning that v54 is available. Just being consistent about pinning it makes it clear that it's intentional and I don't get a warning :)